ext/pdo/pdo_dbh.stub.php

```diff
@@ -37,7 +37,7 @@ public function lastInsertId(?string $name = null) {}
     public function prepare(string $statement, array $driver_options = []) {}
 
     /** @return PDOStatement|false */
-    public function query(string $statement, ?int $fetch_mode = null, mixed ...$fetch_mode_args) {}
+    public function query(string $statement, ?int $fetch_mode = null, mixed $arg1 = null, ?array $constructorArguments = []) {}
```

Comment:
These signatures are intentionally variadic, as the accepted arguments depend on the fetch mode -- it's really an overloaded method (https://www.php.net/pdo.query). E.g. passing the last parameter for a fetch mode that doesn't need this argument will result in an error (which makes the signature as written illegal wrt default values).

Reply:
Do I need to change ZPP to variadic? Because ``fetchAll()`` used formal ZPP params although it was a variadic in the stubs.